styles/base.less

```diff
@@ -1,7 +1,7 @@
 @import "syntax-variables";
 @import "system.less";
 
-atom-text-editor, :host {
+atom-text-editor, atom-text-editor {
```

Comment:
One of these should be removed, you don't need to have the same thing on both sides of a comma.